### PR TITLE
Add severe log in Action.java #3100

### LINKED ISSUE #3100: Add severe log in Action.java
Place a log: log.severe("TEAMMATES accessed using old join link")
in branch where legacy REG key is being used.

Related to #3011, #3081

### PR DESCRIPTION
**Pull request by jarvis57**
Fixes #3100

### CODE REVIEW

File: src/main/java/teammates/ui/controller/Action.java [hunk 1]
```diff
@@ -129,6 +129,7 @@ protected String getRegkeyFromRequest() {
             //TODO: remove this branch on October 15th 2014.
             return getRequestParamValue(Const.ParamsNames.REGKEY_LEGACY);
         } else {
+            log.severe("TEAMMATES accessed using old join link");
```
Comment (kanghj) [inline src/main/java/teammates/ui/controller/Action.java]: Shouldn't the log.severe be in the other branch?

Comment (jarvis57) [inline src/main/java/teammates/ui/controller/Action.java]: I forgot the "legacy". Sorry for the misunderstanding.

**Comment by damithc** [comment]
This PR name is not in the right format.

**Comment by damithc** [comment]
Actually, the PR name format is correct. I didn't realize that a new issue with a different name has been created.

**Comment by damithc** [comment]
@thyageshm I'm getting quite a few 'TEAMMATES accessed using old join link` in the logs. Possibly this branch is also used for something else? In fact looks like many other requests travel though this branch.

**Comment by thyageshm** [comment]
looking at it now

**Comment by thyageshm** [comment]
sir, it is not that the branch is not executed. in fact it is executed everytime the normal regkey is not found. I think the placement of the log is wrong. we should first ensure that the getRequestParamValue(Const.ParamsNames.REGKEY_LEGACY); function is returning a non-null value and then log

**Comment by damithc** [comment]
May be get @jarvis57 to create a new PR with the correct code?

**Comment by jarvis57** [comment]
I am glad to do so sir. So will we still keep it as issue 3100?

**Comment by damithc** [comment]
May be it's better to create a new issue first. That way we can follow the same workflow.

**Comment by jarvis57** [comment]
So please just assign it to me, I will fix it as soon as possible.

**Comment by jarvis57** [comment]
Sir I haven't gotten the issue. Could you create the issue so that I can make the new PR.
(Or I have to create the issue by myself? Because I don't know how to number the issue.)

**Comment by damithc** [comment]
@thyageshm Someone accessed TEAMMATES using an old key today :-)

**Comment by thyageshm** [comment]
ok! I guess we have to keep this for longer :-)

**Pull request merged**